bazel/README.md

```diff
@@ -85,6 +85,9 @@ for how to update or override dependencies.
 
     ### Linux
     On Linux, we recommend using the prebuilt Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html).
+
+    Now we use 11.0.1 version.
```

Comment:
I'd simplify... and concatenate this to the first statement... "from [LLVM...](...), version 11.0.1 or higher."

This change goes out of date immediately when we bump the linux toolchain. On the windows toolchain, we are at 11.1.0.

Reply:
We don't require 11.0.1, I do still use LLVM 9 locally sometime.

Reply:
ping @lizan